wbb/modules/repo.py

```diff
@@ -4,7 +4,7 @@
 
 
 __MODULE__ = "Repo"
-__HELP__ = "/repo - To Get My Github Repository Link " + \
+__HELP__ = "/repo - To Get My Github Repository Link " \
            "And Support Group Link"
 
```

Comment:
You can write it as this to avoid backslash. current one maybe also ok.

```
__HELP__ = ("/repo - To Get My Github Repository Link "
            "And Support Group Link")
```

Reply:
brackets look ugly, i tried